ui/src/tool-panel/km-post/km-post-infobox.tsx

```diff
@@ -191,7 +191,10 @@ const KmPostInfobox: React.FC<KmPostInfoboxProps> = ({
                             />
                             <InfoboxField
                                 label={t('tool-panel.changed')}
-                                value={formatDateShort(kmPostCreatedAndChangedTime.changed)}
+                                value={
+                                    kmPostCreatedAndChangedTime.changed &&
+                                    formatDateShort(kmPostCreatedAndChangedTime.changed)
+                                }
```

Comment:
Päätykös näissä tämän PR:n kolmessa value-chekissä <InfoboxField>-komponentille mahdollisesti value-propsuksi false? Onko siitä haittaa vai pitäiskö näillä olla jokin defaultti?

Reply:
Käytännössä tänne voi päätyä `undefined` kun tuo `changed` on nykyään nullable-propsi. Tällöin kenttä näkyy tyhjänä. Tuo `formatDateShort()` pitää huolen siitä, että jos päästään sinne saakka, niin sieltä tulee ulos joko formatoitu päivämäärä tai empty string.